dev/Projections/CS/Microsoft.Windows.AppNotifications.Projection/Microsoft.Windows.AppNotifications.Projection.csproj

```diff
@@ -39,6 +39,10 @@
       <HintPath>$(OutDir)..\WindowsAppRuntime_DLL\StrippedWinMD\Microsoft.Windows.AppNotifications.winmd</HintPath>
       <IsWinMDFile>true</IsWinMDFile>
     </Reference>
+	  <Reference Include="Microsoft.Windows.AppNotifications.Builder">
```

Comment:
Initially, I wanted to follow the same pattern that other APIs have been following and create a new project for the builder but the builder depends on AppNotification and I couldn't figure out how to make that work with two separate projects.

The thing is anyway, there is no point in using the builder without AppNotification as all it can do is build new AppNotification instances.